pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -311,3 +337,40 @@ func manageAPIServices_v311_00_to_latest(client apiregistrationv1client.APIServi
 
 	return apiServices, nil
 }
+
+var openshiftScheme = runtime.NewScheme()
+
+func init() {
+	if err := openshiftapi.Install(openshiftScheme); err != nil {
+		panic(err)
+	}
+}
+
+func resourceSelectorForCLI(obj runtime.Object) string {
+	groupVersionKind := obj.GetObjectKind().GroupVersionKind()
+	if len(groupVersionKind.Kind) == 0 {
+		if kinds, _, _ := scheme.Scheme.ObjectKinds(obj); len(kinds) > 0 {
+			groupVersionKind = kinds[0]
+		}
+	}
+	if len(groupVersionKind.Kind) == 0 {
+		if kinds, _, _ := openshiftScheme.ObjectKinds(obj); len(kinds) > 0 {
+			groupVersionKind = kinds[0]
+		}
+	}
+	if len(groupVersionKind.Kind) == 0 {
+		groupVersionKind = schema.GroupVersionKind{Kind: "Unknown"}
+	}
+	kind := groupVersionKind.Kind
+	group := groupVersionKind.Group
+	var name string
+	accessor, err := meta.Accessor(obj)
+	if err != nil {
+		name = "unknown"
+	}
+	name = accessor.GetName()
+	if len(group) > 0 {
+		group = "." + group
+	}
+	return kind + group + "/" + name
```

Comment:
@soltysh @sttts i saw this code so many times... last time in event recorder :-)